cassava.cabal

```diff
@@ -101,6 +101,7 @@ Library
     hashable < 1.3,
     scientific >= 0.3.4.7 && < 0.4,
     text < 1.3,
+    transformers >= 0.2 && < 1,
```

Comment:
Why did you use `<1`?

Reply:
Do you have any suggestion? All other dependencies have upper bounds so I thought I should put something.

Reply:
Well, I was wondering what made you use `<1` instead of the more appropriate `<0.6`

Reply:
I was just not sure what it should be. I changed it to 0.6